srcs/SetServer.cpp

```diff
@@ -180,8 +183,9 @@ int Server::recieve_msg(int new_fd, int i)
 	else
 	{
 		std::cout << buf << std::endl;
-		setMessage(buf);
 		setClientId(new_fd);
+		setMessage(buf);
+		// we start parsing here
```

Comment:
this is the part where we start the parsing, we receive the message and the file descriptor as parameter